src/util.ts

```diff
@@ -0,0 +1,19 @@
+
+export const anArrayOf = <A>(n: number, a: A): A[] => Array.from(Array(n)).map(() => a)
+
+/**
+ * Given two arrays [a1, a2, ...], [b1, b2, ...] make an array
```

Comment:
This util content should be moved into the util.ts file in `src\irsForms` rather than being in the `src` directory.

Reply:
I put them here because they are so generic that they aren't specific or related to what our project is doing. Whereas the functions in irsforms are specific to displaying data in those forms.